src/Summoner/Template/Stack.hs

```diff
@@ -16,31 +16,24 @@ stackFiles Settings{..} = map createStackYaml settingsTestedVersions
  where
     -- create @stack.yaml@ file with LTS corresponding to specified ghc version
     createStackYaml :: GhcVer -> TreeFs
-    createStackYaml ghcV = let ver = case ghcV of
-                                  Ghc843 -> ""
-                                  _      -> "-" <> showGhcVer ghcV
-        in stackYaml ver (latestLts ghcV) (baseVer ghcV)
+    createStackYaml ghcV = File (toString $ "stack" <> ver <> ".yaml")
+        $ "resolver: lts-" <> latestLts ghcV <> extraDeps <> ghcOpts
       where
-        stackYaml :: Text -> Text -> Text -> TreeFs
-        stackYaml ghc lts baseV = File (toString $ "stack" <> ghc <> ".yaml")
-            [text|
-            resolver: lts-${lts}
+        ver :: Text
+        ver = case ghcV of
+            Ghc843 -> ""
```

Comment:
At one hand I feel that in order to be more resilent to refactoring this should be something like `if ghcV == defaultGHC then ...` but on the other hand it might complicate support of default ghc for `stack` and `cabal` when the default is different (like now: `stack` default is GHC-8.4.3 and cabal could have GHC 8.6.1)